pubsub_client.gemspec

```diff
@@ -22,9 +22,11 @@ Gem::Specification.new do |spec|
   end
   spec.require_paths = ['lib']
 
+  spec.add_runtime_dependency 'google-cloud-pubsub', '~> 2.0'
+  spec.add_runtime_dependency 'activesupport'
```

Comment:
Can this be a development dep? Outside of the Rake task to build and publish, the gem itself doesn't require `activesupport`.

Reply:
i think activesupport is needed for the rake command.

https://github.com/apartmentlist/pubsub_client/pull/23/files#diff-af9e1062d1a621c40530bb7bda1d331db8a9298c9b4b32ad9d4b31c2ac319a39R11

Reply:
Sounds good, I'll leave it in.